### DISCUSSION TITLE
Subscribe to "download" event

### DISCUSSION THREAD

**Discussion by constascgcg** [cypress-io/cypress#17848] [Questions and Help]
Hi team,

Is there a way to subscribe to the "download" event shown in Cypress runner and get the text from it (similar to what we do with cy.on() for, for example, "window:confirm" event?
![image](https://user-images.githubusercontent.com/59094691/130394805-9a111a56-416f-4291-b965-f2d2ff4cb3b9.png)
console:
![image](https://user-images.githubusercontent.com/59094691/130394815-4dfb4945-feb8-48d4-a627-49f04b3ff543.png)

**Comment by the0neWhoKnocks** [comment]
Also interested in this. Mainly I need the `Download url` since I generate a dynamic file name based on the current time. Poked around in the source code for a bit, but haven't found what actually dispatches that event.

Can any maintainers please provide some guidance?
